rpc_jobs/dummy_pipeline.yml

```diff
@@ -19,10 +19,10 @@
       - '{trigger}-Dep-Update_{repo_name}-{branch}'
 
 - project:
```

Comment:
This should probably be added to a separate file, its purpose is a bit confusing in this dummy pipeline file.

Reply:
Moved into `rpc_releases.yml`. I gave it the `rpc_` prefix because we already have a file called `release.yml`. I figured I'd leave `release.yml` as-s since we'll eventually remove that project once the pipeline is fully operational.